docs/blockchain/repo/tlbc.md

```diff
@@ -0,0 +1,315 @@
+---
```

Comment:
These files are generated right. This should be written somewhere, best in the folder or file